.github/workflows/sycl-linux-precommit.yml

```diff
@@ -89,6 +89,13 @@ jobs:
             reset_gpu: true
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
             extra_lit_opts: --param gpu-intel-gen12=True
+          - name: E2E tests on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --group-add 109 --device=/dev/dri --privileged --cap-add SYS_ADMIN
```

Comment:
do you know why we need `--group-add 109` here but not for DG2/Gen12?

Reply:
PVC runners has a group named "render" (id:109), which is required to access PVC cards. Honestly, I don't know why this group is only on PVC and not on other machines.